### PR TITLE
Toggle button

### PR DESCRIPTION
**Pull request by FX196**

### CODE REVIEW

File: webapp/main.py [hunk 1]
```diff
@@ -411,6 +413,17 @@ def poll():
...
+@app.route('/toggle-server')
+def toggleServer():
+    global current_service_address
+    if current_service_address == compute_service_address:
+        current_service_address = dev_service_address
+        return "development"
+    else:
+        current_service_address = compute_service_address
+        return "production"
+
```
Comment (FX196) [inline webapp/main.py]: Can you store the `current_service_address` variable inside `session`? Right now the `current_service_address` is global, meaning that if one user uses the button, it'll change the service address for everyone. `session` is a cookie variable, meaning it's tied to individual users/browsers. Check out lines 95-97 for usage

**Review by FX196 — COMMENTED** [review]
Also storing the current service address is fine, but perhaps using a boolean would be more elegant

File: webapp/main.py (hunk 1 above)
Comment (nhuang321) [reply webapp/main.py]: ok, this should be fixed now

**Review by FX196 — COMMENTED** [review]
Code looks good to me, has this been tested yet?

**Pull request merged**